core/src/main/resources/hudson/model/View/configure.jelly

```diff
@@ -55,12 +55,8 @@ THE SOFTWARE.
         <!-- view property configurations -->
         <f:descriptorList descriptors="${it.getVisiblePropertyDescriptors()}" instances="${it.properties}" />
 
-        <f:bottomButtonBar>
-          <f:submit value="${%OK}" />
```

Comment:
This effectively changes the label `OK` to `Save`. But I don't think there was any good reason for this label to differ from other screens.